course_catalog/etl/video.py

```diff
@@ -0,0 +1,66 @@
+"""video catalog ETL"""
+
+from django.conf import settings
+from googleapiclient.discovery import build
+from django.conf import settings
+
+class YouTubeApi:
```

Comment:
We usually try to avoid classes where possible and stick to regular functions, could you refactor this to reflect that?